src/main/java/uk/ac/cam/cl/dtg/segue/api/managers/ExternalAccountManager.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright 2021 James Sharkey
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ * 		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.cam.cl.dtg.segue.api.managers;
+
+import com.google.inject.Inject;
+import com.mailjet.client.errors.MailjetClientCommunicationException;
+import com.mailjet.client.errors.MailjetException;
+import com.mailjet.client.errors.MailjetRateLimitException;
+import org.json.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.ac.cam.cl.dtg.segue.dao.SegueDatabaseException;
+import uk.ac.cam.cl.dtg.segue.dao.users.IExternalAccountDataManager;
+import uk.ac.cam.cl.dtg.segue.dos.users.EmailVerificationStatus;
+import uk.ac.cam.cl.dtg.segue.dos.users.UserExternalAccountChanges;
+import uk.ac.cam.cl.dtg.util.email.MailJetApiClientWrapper;
+import uk.ac.cam.cl.dtg.util.email.MailJetSubscriptionAction;
+
+import java.util.List;
+
+public class ExternalAccountManager {
```

Comment:
In terms of function could be worth adding a bit of JavaDoc to explain what external accounts this thing is intended to manage. Is it really just email or do we think it would be easily extended to do other stuff in the future?